NuGet.config

```diff
@@ -13,6 +13,14 @@
     <add key="dotnet8" value="https://dnceng.pkgs.visualstudio.com/public/_packaging/dotnet8/nuget/v3/index.json" />
     <add key="dotnet8-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8-transport/nuget/v3/index.json" />
     <add key="dotnet9" value="https://dnceng.pkgs.visualstudio.com/public/_packaging/dotnet9/nuget/v3/index.json" />
+    <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
```

Comment:
is it a temporary thing?
I recall we had some limitations for using this feed in msbuild repo

Reply:
It's intended to be permanent.

Reply:
@rainersigwald to draw your attention here

Reply:
The PSM configuration doesn't exactly say that: it says that `Microsoft.VisualStudio.OpenTelemetry*` must come from `vs-impl`, not that `vs-impl` can only be used for that package.

We need to be careful here--have you talked to the VMR/sourcebuild folks about the dependency?

Reply:
> The PSM configuration doesn't exactly say that: it says that Microsoft.VisualStudio.OpenTelemetry* must come from vs-impl, not that vs-impl can only be used for that package.
Thanks, I misunderstood that. Is it possible to do? From a quick look at docs I guess no ☹️ 

We're in contact and it does not impact sourcebuild, because it's references are gated only for .NET Framework in Microsoft.Build.Framework.csproj, which is not a part of sourcebuild.

There is no way to condition on that in NuGet.config though.

Reply:
It's in dotnet-tools feed, removed vs-impl.